### DISCUSSION TITLE
Support for multiple remote paths in RemoteTransform2D

### DISCUSSION THREAD

**Discussion by CodeLazier** [godotengine/godot-proposals#6778] [2D]
I have multiple different nodes that need to be synchronized to the player, currently RemoteTransform2D can only synchronize one Node2D node, I propose that RemoteTransform2D "Remote Path" be changed to Array[Node2D], so that it is convenient to synchronize multiple nodes and improve performance.

**Comment by AThousandShips** [comment]
Is there anything preventing you from parenting those nodes to a single node that's remotely controlled?

**Reply by CodeLazier** [reply]
Probably a fear of breaking the y-sort relationship

**Reply by AThousandShips** [reply]
That makes sense, but would only apply to 2D as a limitation

**Reply by CodeLazier** [reply]
Yes, for this rechanged title
